scalafmt-core/shared/src/main/scala/org/scalafmt/config/ScalafmtRunner.scala

```diff
@@ -87,6 +81,7 @@ object ScalafmtRunner {
     val scala213 = Scala213
       .withAllowTrailingCommas(true)
     val default = scala213
+      .withAllowTraitParameters(true) // not true but backwards compatible
```

Comment:
Why do we need it? Aren't trait params only available from Scala 3 ?

Reply:
we have two options:
- keep this
- remove this test: https://github.com/scalameta/scalafmt/blob/master/scalafmt-tests/src/test/resources/unit/Dialect.stat#L20
thoughts?

Reply:
I would remove the tests, although I am not sure why it was needed in the first place.

Reply:
will do.